scripts/core/editor3/components/Editor3.jsx

```diff
@@ -94,18 +94,21 @@ export class Editor3Component extends React.Component {
     /**
      * @ngdoc method
      * @name Editor3#onDragOver
-     * @returns {Boolean} Returns true if the item is permitted.
+     * @returns {Boolean} Returns true if the item is not permitted.
      * @description Checks if the dragged over item is allowed.
      */
     onDragOver(e) {
+        const {editorFormat, readOnly, singleLine} = this.props;
         const mediaType = e.originalEvent.dataTransfer.types[0] || '';
-
-        return [
+        const goodFile = [
```

Comment:
I would name this `isValidMedia` because it checks the validity of the media type, not a file.